tests/phpt/cl/178_fail_pass_class_to_hll_stats.php

```diff
@@ -0,0 +1,8 @@
+@kphp_should_fail
+/conversion from A to mixed is forbidden/
+<?php
+class A {}
+function demo() {
+  var_dump(vk_stats_hll_merge(new A)); // подходит любая функция из functions.txt, принимающая var
```

Comment:
```suggestion
  var_dump(vk_stats_hll_merge(new A)); // подходит любая функция из functions.txt, принимающая mixed
```

Reply:
lol, copy-paste, I think I'll just remove this comment.